data/json/items/comestibles/carnivore.json

```diff
@@ -785,6 +857,24 @@
     "calories": 54,
     "vitamins": [ [ "vitC", 0 ], [ "calcium", 1 ], [ "iron", 25 ] ]
   },
+  {
+    "id": "human_kidney",
+    "type": "COMESTIBLE",
+    "copy-from": "kidney",
+    "looks_like": "kidney",
+    "name": { "str": "raw human kidney" },
+    "description": "Kidney from a human, which appears to have been extracted too crudely to ever be useful as a transplant.",
+    "material": [ "hflesh" ]
+  },
+  {
+    "id": "demihuman_kidney",
+    "type": "COMESTIBLE",
+    "copy-from": "kidney",
+    "looks_like": "kidney",
+    "name": { "str": "raw demihuman kidney" },
+    "description": "Kidney extracted from a demihuman being.",
```

Comment:
```suggestion
    "description": "This is probably a kidney extracted from a demihuman being.",
```

Reply:
As far as I understand, people versed in butchering are able to identify organs from different species without too much trouble. Unless demihumans are supposed to come from another dimension, and thus won't be of an Earth evolutionary origin, identifying their organs ought to be fairly straightforward.